repo/fsrepo/fsrepo.go

```diff
@@ -142,7 +142,7 @@ func open(repoPath string) (repo.Repo, error) {
 	}()
 
 	// Check version, and error out if not matching
-	ver, err := mfsr.RepoPath(r.path).Version()
+	ver, err := migrations.RepoVersion(r.path)
```

Comment:
Why did we change this pattern, just less indirection?

Reply:
There were multiple implementations of this same code, here and in ipfs-update.  I chose the API that I felt was the simplest and most readable.

Reply:
Fair enough, I mostly wanted to know why we did the change. This way does a little more repeated work in exchange for a slightly cleaner API which SGTM.